libs/styles/styles.css

```diff
@@ -124,6 +124,13 @@
   --icon-size-s: 32px;
   --icon-size-xs: 24px;
   --icon-size-xxs: 16px;
+
+    /* border-radius sizes */
+    --border-radius-s: 4px;
+    --border-radius-l: 8px;
+    --border-radius-xl: 16px;
+    --full-border-radius: 50%;
+    --no-border-radius: 0;
```

Comment:
I'd argue that the meaning of 'no border radius' will always be '0', so I'd save on a few dozen characters and just use `0` where needed.

Reply:
This was to keep similarity between the class names used by the authors and In case if the same is used in some other block in future. But you are correct, we can directly use 0.